cli/cmd/get.go

```diff
@@ -42,7 +42,7 @@ import (
 )
 
 const (
-	maxClassesToDisplay = 20
+	maxClassesToDisplay = 75
```

Comment:
maybe this should be moved to a package that can be imported here and in `workloads`, to have a single source of truth?